packages/create-fuels/src/lib/setupProgram.ts

```diff
@@ -2,9 +2,9 @@ import { Command } from 'commander';
 
 import packageJson from '../../package.json';
 
-export type Template = 'nextjs';
-export const templates: Set<Template> = new Set(['nextjs']);
-export const defaultTemplate: Template = 'nextjs';
+export type Template = 'nextjs' | 'vite';
```

Comment:
I wonder if we still want to enforce these types? Because as we move to support more templates just having to update these continuously doesn't seem to outweigh the benefits of maintaining these types.

Reply:
They are super helpful in some utility functions that are dependent on the template that's being used. I don't think we are going to be supporting too many more templates so this should be fine in my opinioin